tests/AuthService.test.js

```diff
@@ -1,12 +1,78 @@
-const AuthService = require ("../core/services/AuthService.js");
+global.AuthService = require ("../core/services/AuthService.js");
+global._ = require("lodash");
 
 describe ("AuthService", () => {
-  it ("Password encrypt and check", () =>{
+  it("Password encrypt and check", () => {
     const passwords = ["rrr", "fdgfdg", 1234, {a: "b"}];
     for (let password of passwords) {
       (typeof(password) === "string")
       ? expect(AuthService.checkPassword(password, AuthService.encryptPassword(password))).toBeTruthy()
       : expect(() => {AuthService.encryptPassword(password)}).toThrow();
     }
-  })
+  });
+  it("Token generate and check", () => {
+    const operator = {
+      id: 23,
+      email: "example@gmail.com",
+      passwordHash: AuthService.encryptPassword("heDK4src4w")
+    };
+    const token = AuthService.generateToken(operator);
+    const decodedToken = AuthService.checkAndDecodeToken(token);
+    expect(decodedToken.id).toBe(operator.id);
+    expect(decodedToken.email).toBe(operator.email);
+    expect(decodedToken.exp).toBeGreaterThan(Date.now()/1000);
```

Comment:
Another thing – you don't check the expiration of your token. So, you expect that after some seconds your `checkAndDecodeToken` function should throw an exception – but you can't be sure it does until you write a test for it. So, your test should create a token, test it, wait for required time using `setTimeout`, and then test it again expecting the error (Example of how to expect error you can find in `Adapter.test.js`). To make the test faster, you can set smaller token lifetime value inside `config/test.js`